src/scales/quantitativeScale.ts

```diff
@@ -115,8 +114,8 @@ module Plottable {
       }
     }
 
-    public _defaultExtent(): any[] {
-      return [0, 1];
+    public _defaultExtent(): D[] {
+      throw Error("The quantitative scale itself does not have a default extent");
```

Comment:
This sorta basically disallows QuantitativeScales to be instantiated and used on its own.  Is this ok?